nixos/modules/services/databases/tigerbeetle.md

```diff
@@ -35,3 +35,10 @@ Note that the TigerBeetle module won't open any firewall ports automatically, so
 
 A complete list of options for TigerBeetle can be found [here](#opt-services.tigerbeetle.enable).
 
+## Upgrading {#module-services-tigerbeetle-upgrading}
+
+Usually, TigerBeetle's [upgrade process](https://docs.tigerbeetle.com/operating/upgrading) only requires replacing the binary used for the servers.
+This is not directly possible with NixOS since the new binary will be located at a different place in the Nix store.
+
+However, since TigerBeetle is managed through systemd on NixOS, the only action you need to take when upgrading is to make sure the version of TigerBeetle you're upgrading to supports upgrades from the version you're currently running.
```

Comment:
I think this requires [building TigerBeetle binaries using the `multiversion` flag](https://github.com/tigerbeetle/tigerbeetle/blob/198f73517e41256f32757befa6a1b381f614d13e/build.zig#L90) in order to include older versions of the binary, which is not currently done in Nixpkgs.

Reply:
This is no longer the case after https://github.com/NixOS/nixpkgs/pull/332867